README.md

```diff
@@ -14,6 +14,60 @@ Latest DockerHub Images: https://hub.docker.com/u/pangeo
 
 *Click on the image name in the table above for a current list of installed packages and versions*
 
+These images are meant to be used with [Pangeo Cloud](https://pangeo.io/cloud.html) JupyterHub environments, and provide comprehensive sets Python packages for geospatial analysis that are commonly used for scientific research. Package versions are tracked for reproducibility (see links in table above).
+
+### How to use the base-image with a Pangeo Binder
+A major use-case for these images is running an ephemeral server on the Cloud with BinderHub. Anyone can launch a server running the latest-and-greatest `pangeo-notebook` image with the following URLs for running in GCP us-central1 or AWS us-west-2 respectively:
+
+```
+https://binder.pangeo.io/v2/gh/pangeo-data/pangeo-docker-images/HEAD?urlpath=lab
```

Comment:
I don't understand this part. Whenever I use these images, I point to a passthrough repo like https://github.com/pangeo-gallery/default-binder

If I point a binder at this repo, I think it will use this environment? https://github.com/pangeo-data/pangeo-docker-images/blob/master/.binder

How does that lead to launching `pangeo-notebook`?

Reply:
I'm not sure I 100% understand your concern here, but binder builds from the dockerfile if available, so it'll use the environment defined by https://github.com/pangeo-data/pangeo-docker-images

Reply:
Which docker file though? Binder will look for a folder named `binder/` or `.binder/`.

---

Ok, I think I get it! We are using a soft link! https://github.com/pangeo-data/pangeo-docker-images/blob/master/.binder

Reply:
> Ok, I think I get it! We are using a soft link! https://github.com/pangeo-data/pangeo-docker-images/blob/master/.binder 

Exactly! I wasn't sure it would work, but it does :)